lib/MusicBrainz/Server/Data/Artwork.pm

```diff
@@ -145,6 +145,20 @@ sub find_front_cover_by_release
     return \@artwork;
 }
 
+sub find_count_by_release
+{
+    my ($self, $release_id) = @_;
+
+    return unless $release_id; # nothing to do
+    my $query = "SELECT count(*)
+        FROM cover_art_archive.index_listing
+        JOIN cover_art_archive.cover_art
```

Comment:
Is that `JOIN` needed?

Reply:
Guess not, actually, now that I properly looked into index_listing!